dataactbroker/handlers/fileHandler.py

```diff
@@ -479,9 +498,134 @@ def checkD1File(self, submission_id=None):
 
         return JsonResponse.create(StatusCode.OK, response)
 
+    def loadGenerateRequest(self):
```

Comment:
Not a big deal, but I found this method name confusing. Would something like `getSubmissionAndFileType` be too on the nose?